res/values-pt/strings.xml

```diff
@@ -1,49 +1,53 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-  <string name="settings_activity_label">Configurações Unexpected Keyboard</string>
```

Comment:
The xml files are auto formatted with `xmllint`, that's why there are often very long lines. To avoid the indentation from changing again next time, I updated your PR.